client/src/test/java/pl/edu/agh/RaftClientTest.java

```diff
@@ -0,0 +1,141 @@
+package pl.edu.agh;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.junit.Before;
+import org.junit.Test;
+import pl.edu.agh.utils.SocketAddressUtils;
+import pl.edu.agh.utils.ThreadUtils;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by Andrzej on 2016-11-11.
```

Comment:
I will remove all those comments but it's mean of you to make me remove auto-generated comments xD